src/qibolab/platform.py

```diff
@@ -201,6 +198,28 @@ def update(self, updates: dict):
                 elif par == "classifiers_hpars":
                     self.qubits[qubit].classifiers_hpars = value
 
+                # Flux resonator spectroscopy
+                elif par == "flux_to_bias":
+                    self.qubits[qubit].flux_to_bias = float(value)
+
+                elif par == "asymmetry":
+                    self.qubits[qubit].asymmetry = float(value)
+
+                elif par == "readout_coupling":
+                    self.qubits[qubit].g = float(value)
+
+                elif par == "brf":
+                    self.qubits[qubit].brf = float(value)
+
+                elif par == "ssf_brf":
+                    self.qubits[qubit].ssf_brf = float(value)
+
+                elif par == "Ec":
+                    self.qubits[qubit].Ec = float(value)
+
+                elif par == "Ej":
+                    self.qubits[qubit].Ej = float(value)
+
```

Comment:
To avoid creating conflicts with https://github.com/qiboteam/qibolab/pull/596 you can remove those lines.

Reply:
I have removed these lines. @andrea-pasquale let me know when qibocal is ready to update the parameters, otherwise we can not use the flux routines

Reply:
I've opened the PR in qibocal https://github.com/qiboteam/qibocal/pull/538 @DavidSarlle